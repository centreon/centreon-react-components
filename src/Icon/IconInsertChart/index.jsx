import React from 'react';
import InsertChart from '@material-ui/icons/InsertChart';
import MaterialIcon from '../MaterialIcon';

function IconInsertChart(props) {
  return (
    <MaterialIcon {...props}>
      <InsertChart />
    </MaterialIcon>
  );
}

export default IconInsertChart;
