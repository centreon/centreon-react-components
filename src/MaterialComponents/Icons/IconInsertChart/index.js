/* eslint-disable react/jsx-filename-extension */

import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import InsertChart from '@material-ui/icons/InsertChart';

const useStyles = makeStyles(() => ({
  root: {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
  },
  icon: {
    color: '#707070',
    cursor: 'pointer',
  },
  iconWrap: {
    display: 'inline-block',
    verticalAlign: 'middle',
    height: 24,
  },
}));

function IconInsertChart({ ...rest }) {
  const classes = useStyles();

  return (
    <span {...rest} className={classes.iconWrap}>
      <InsertChart className={classes.icon} />
    </span>
  );
}

export default IconInsertChart;
