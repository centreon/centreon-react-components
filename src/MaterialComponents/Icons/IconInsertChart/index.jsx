import React from 'react';
import InsertChart from '@material-ui/icons/InsertChart';
import MaterialIcon from '../MaterialIcon';

function IconInsertChart() {
  return (
    <MaterialIcon>
      <InsertChart />
    </MaterialIcon>
  );
}

export default IconInsertChart;
