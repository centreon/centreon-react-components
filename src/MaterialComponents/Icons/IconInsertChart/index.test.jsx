/* eslint-disable no-undef */

import React from 'react';
import { create } from 'react-test-renderer';
import IconInsertChart from '.';

describe('IconInsertChart', () => {
  it('renders', () => {
    const wrapper = create(<IconInsertChart />);

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
