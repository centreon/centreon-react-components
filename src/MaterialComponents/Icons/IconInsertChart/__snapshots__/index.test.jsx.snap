// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconInsertChart renders 1`] = `
.emotion-0 {
  display: inline-block;
  vertical-align: middle;
  height: 24px;
  cursor: pointer;
  color: #707070;
}

<span
  className="emotion-0"
>
  <svg
    aria-hidden="true"
    className="MuiSvgIcon-root"
    focusable="false"
    role="presentation"
    viewBox="0 0 24 24"
  >
    <path
      d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z"
    />
    <path
      d="M0 0h24v24H0z"
      fill="none"
    />
  </svg>
</span>
`;
